src/components/game/category-overview.tsx

```diff
@@ -117,6 +119,24 @@ export const CategoryOverview = ({
                     );
                 })}
             </tbody>
+            <br />
+            {categories.length > 5 ? (
+                showAllCategories === 5 ? (
+                    <button
+                        name="Show More"
+                        onClick={() => setShowAllCategories(100)}
```

Comment:
In this scenario would this mean that we can only support up to a maximum of 100 categories?

Reply:
Yes, admittedly not a clean solution and more of a "it works" solution. Now that I think about it more, this probably makes more sense. 
```suggestion
                        onClick={() => setShowAllCategories(category.length)}
```